.vscodeignore

```diff
@@ -1,3 +1,32 @@
+# Allowed:
+# build/
+# resources/
+# images/
+# LICENSE
+
+.env
+.env.example
+.eslintignore
+.mocharc.js
+.nvmrc
+.prettierignore
+CODEOWNERS
+HOMEPAGE.md
+changelog.md
+nightlyver.sh
+package.json
```

Comment:
Interested to see how this might impact the marketplace page after we do a release

Reply:
True, that's not something I could test - however, looks like `readme` and `changelog` are pulled in regardless, as well as package.json:
![image](https://github.com/user-attachments/assets/99dc09d9-2be4-45f3-a7bd-f6c6ce88dd10)

I think most of the contents of the marketplace page should be built on links - so it might be fine? But we'll only see after the 1st nightly